src/common/components/modal/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import { Icon, IconButton } from "~/common/components/index";
+import { IconName } from "~/common/enums/index";
+import { useModal } from "~/common/hooks/index";
+
+import styles from "./styles.module.scss";
+
+type ModalProperties = {
+	children: React.ReactNode;
+	footer?: React.ReactNode;
+	isOpen: boolean;
+	onClose: () => void;
+	title: string;
```

Comment:
maybe the name is optional? For example  page 'moderatorx-review' dosen't has this title